packages/smooth_app/android/app/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
         targetSdkVersion 30
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        multiDexEnabled true
```

Comment:
It's not a good sign if we need to turn the multidex on: we're getting fatter and fatter.

Reply:
Maybe we are able to finetune some minify settings but yes looks like it